docs/faqs/max_execution_time.rst

```diff
@@ -86,4 +85,7 @@ Other limitations
 ***************************
 
 - Programs cannot exceed 750KB in size.
-- Inputs to jobs cannot exceed 64MB in size.
\ No newline at end of file
+- Inputs to jobs cannot exceed 64MB in size.
+- Open users are limited to 10 minutes of job execution time per month.  This is the time that the QPU
+   complex (including control software, control electronics, QPU, and so on) is engaged in
+   processing the job. 
```

Comment:
Can we add some words about one can find the remaining quota on the IQP dashboard?